code/game/objects/items/storage/toolbox.dm

```diff
@@ -252,7 +252,7 @@
 	if(!istype(T, /obj/item/stack/tile/plasteel))
 		..()
 		return
-	if(!is_type_in_list(src, allowed_toolbox))
+	if(!is_type_in_list(src, allowed_toolbox) && (type != /obj/item/storage/toolbox))

```

Comment:
Why not just add the type to the list?

Reply:
is_type_in_list would include all the subtypes, which isn't what we want here.

Reply:
likewise, we couldn't do a specific typecache since we'd want to use all subtypes of the previously defined toolboxes.

Reply:
`- subtypesof(/obj/item/storage/toolbox)`?

Reply:
?

Reply:
can't you manipulate typecaches like such or do I need my afternoon coffee?

Reply:
but you want things that -are- subtypes of /toolbox, just not -all- the subtypes of /toolbox.

Reply:
😭